includes/Story_Post_Type.php

```diff
@@ -147,6 +148,26 @@ public function init() {
 		add_filter( 'googlesitekit_amp_gtag_opt', [ $this, 'filter_site_kit_gtag_opt' ] );
 	}
 
+	/**
+	 * Base64 encode the svg file for post tpye icon.
+	 *
+	 * @param string $icon Name of icon file.
+	 *
+	 * @return bool|string Returns false is the file doesn't exist.
+	 */
+	public function get_post_type_icon( $icon ) {
+		$path = WEBSTORIES_PLUGIN_DIR_PATH . 'assets/images/' . $icon;
+		if ( ! is_readable( $path ) ) {
+			return false;
+		}
+		$icon_contents = file_get_contents( $path ); // phpcs:ignore WordPressVIPMinimum.Performance.FetchingRemoteData.FileGetContentsUnknown
+		if ( ! $icon_contents ) {
+			return false;
+		}
+
+		return 'data:image/svg+xml;base64,' . base64_encode( $icon_contents );
+	}
```

Comment:
```suggestion
	/**
	 * Returns the post type icon.
	 *
	 * Returns the Base64-encoded icon from assets/images/stories.svg.
	 *
	 * @return string Post type icon.
	 */
	public function get_post_type_icon( $icon ) {
		return 'data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNjMiIGhlaWdodD0iNTUiIGZpbGw9Im5vbmUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHBhdGggZD0iTTAgOGg0djM5SDBWOHpNNTkgOGg0djM5aC00Vjh6TTUwIDBIMTN2NTVoMzdWMHoiIGZpbGw9ImN1cnJlbnRDb2xvciIvPjwvc3ZnPg==';
	}
```

Reply:
Granted, it looks cryptic at first, but this is an acceptable tradeoff.

Reply:
New icon as mentioned on the issue:

```
data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGVuYWJsZS1iYWNrZ3JvdW5kPSJuZXcgMCAwIDI0IDI0IiBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCI+PGc+PHJlY3QgZmlsbD0ibm9uZSIgaGVpZ2h0PSIyNCIgd2lkdGg9IjI0Ii8+PC9nPjxnPjxnLz48Zz48cmVjdCBoZWlnaHQ9IjE1IiB3aWR0aD0iMTAiIHg9IjciIHk9IjQiLz48cmVjdCBoZWlnaHQ9IjExIiB3aWR0aD0iMiIgeD0iMyIgeT0iNiIvPjxyZWN0IGhlaWdodD0iMTEiIHdpZHRoPSIyIiB4PSIxOSIgeT0iNiIvPjwvZz48L2c+PC9zdmc+
```

Reply:
This was my original idea, but it really makes changing this icon hard in the future. Reading a file into memory and base64 encoding really isn't not a lot of overhead.

Reply:
I see your point, but this icon will almost certainly never change again in the future though, so this would keep things much simpler.